### PR TITLE
Setup database

### PR DESCRIPTION
**Pull request by anniviz**

### CODE REVIEW

File: server/models/SongFull.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const mongoose = require('mongoose')
+
```
Comment (srmnq) [inline server/models/SongFull.js]: probably unnecessary

Comment (srmnq) [reply server/models/SongFull.js]: the whole file

File: server/package.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    "start": "nodemon --inspect server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "Annemarie Wisbar",
+  "license": "ISC",
+  "dependencies": {
+    "express": "^4.17.1",
+    "mongoose": "^5.7.13",
+    "npm-run-all": "^4.1.5"
```
Comment (srmnq) [inline server/package.json]: im guessing you could delete this

File: src/services.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  // console.log(res.json())
+  return res.json()
+  // return fetch('/songs').then(res => res.json())
+}
+
+// export function postCard(card) {
+//   return fetch('/cards', {
+//     method: 'POST',
+//     body: JSON.stringify(card),
+//     headers: {
```
Comment (srmnq) [inline src/services.js]: delete comments

**Review by srmnq — APPROVED** [review]
awesome!!!! very impressed!

File: server/models/SongFull.js (hunk 1 above)
Comment (anniviz) [reply server/models/SongFull.js]: done

File: server/package.json (hunk 2 above)
Comment (anniviz) [reply server/package.json]: done

File: src/services.js (hunk 3 above)
Comment (anniviz) [reply src/services.js]: done

**Pull request merged**